investment_tracking/views.py

```diff
@@ -1,3 +1,26 @@
-from django.shortcuts import render
+from django.shortcuts import get_object_or_404
+from rest_framework.response import Response
+from rest_framework.permissions import IsAuthenticated
+from rest_framework import status
+from .models import InvestmentTracking
+from startups.models import StartUpProfile
+from django.db.utils import IntegrityError
+from rest_framework.views import APIView
+from investors.models import InvestorProfile
 
-# Create your views here.
+class InvestmentTrackingSaveView(APIView):
+    """
```

Comment:
Maybe, as you did in startups/views.py, we should also use generics? For example:
class InvestmentTrackingSaveView(generics.CreateAPIView)

Reply:
I think about this. But I don't know which of the realizations will be better. I need to check creation on my own, that's why I implement it this way. But, I can implement these with generics.CreateAPIView and add some checking logic in def perform_create. But I will search more about these two implementations.